pkg/controller/flink/flink.go

```diff
@@ -457,7 +454,7 @@ func (f *Controller) FindExternalizedCheckpoint(ctx context.Context, application
 		return "", nil
 	}
 
-	if time.Since(time.Unix(checkpoint.TriggerTimestamp, 0)) > maxRestoreCheckpointAge {
+	if time.Since(time.Unix(checkpoint.TriggerTimestamp, 0)) > (application.Spec.MaxCheckpointRestoreAgeSec * time.Second) {
```

Comment:
We will need a default value for when this is not set. See https://github.com/lyft/flinkk8soperator/blob/master/pkg/controller/flink/config.go#L38 for an example of this.

Reply:
Set the default to 1 hour.